components/AppHeader.vue

```diff
@@ -201,9 +217,27 @@ export default class AppHeader extends Vue {
         wallet: accounts[0].address,
         method,
       })
-      return this.initWallet(connection)
+      await this.initWallet(connection)
+      if (!this.currentAddress || !this.signer) return
```

Comment:
Throw some error here for ui display?

Reply:
[🥅 Throw connection error](https://github.com/likecoin/app-like-co/pull/474/commits/cc9826077540dfdcd12ebe0f57470e7419f8c922)